jetty-websocket/websocket-common/src/main/java/org/eclipse/jetty/websocket/common/extensions/WebSocketExtensionFactory.java

```diff
@@ -49,7 +51,7 @@
 
     public WebSocketExtensionFactory(WebSocketContainerScope container)
     {
-        lifecycle = new ContainerLifeCycle();
+        containerLifeCycle = new ContainerLifeCycle();
```

Comment:
You still need to override the toString on the ContainerLifeCycle as otherwise this class itself will not appear in any dumps.